github/Repository.py

```diff
@@ -4222,6 +4225,29 @@ def update_custom_properties(self, properties: dict[str, None | str | list]) ->
         }
         self._requester.requestJsonAndCheck("PATCH", url, input=patch_parameters)
 
+    def attach_security_config(self, id: int) -> None:
```

Comment:
same naming as in `Organization`

Reply:
So I made the requested change here, but IMHO the `to_repository` and `from_repository` here are a touch redundant.

Example code (actually from the unit test):

```
repo.attach_security_config_to_repository(id=1234)
```

vs

```
repo.attach_security_config(id=1234)
```

Ultimately I defer to whatever you prefer, but just thought I'd point it out.